python/openvino_tokenizers/constants.py

```diff
@@ -40,3 +40,5 @@ class UTF8ReplaceMode(Enum):
     
     def __str__(self):
         return self.value
+
+SPECIAL_TOKENS_STATE_NAME = "ADD_SPECIAL_TOKENS_VAL"
```

Comment:
should it be named as C++ property? because people can change states via States API and they expect names to be aligned

Reply:
moved this stateful pass to c++, so this variable is removed from here